app-crypt/efitools/Manifest

```diff
@@ -1 +1,2 @@
 DIST efitools-1.7.0.tar.bz2 96042 BLAKE2B b0042df906282d878b1400c359c1eece24588f5df7ab753b62f5f0b95e97467cfaa17e04a2ef40a7ec85155c4d13bd438dd94471e0ec98ab23030dd54e7d4214 SHA512 1e839c5565b0a113569cf47bf98d656875f9a127b481287a553e8b1276969021e1a39a730b9ede3852e15d1ac045db7ec64767f5ac71a24567236c0938697ccb
```

Comment:
Commit message: `Closes` tags should come together, without empty lines in between.

Reply:
Seems to be produced by git, when I do "git commit -m "foo" -m "bar" -m "baz".
But anyway, as I am using now repoman, that will be fixed with the next commit.

Thanks!